docs/bugs.json

```diff
@@ -1,4 +1,17 @@
 [
+    {
+        "uid": "SOL-2023-1",
+        "name": "MissingSideEffectsOnSelectorAccessBug",
+        "summary": "Accessing the ``.selector`` member on complex expressions leaves the expression unevaluated in the legacy code generation.",
+        "description": "When accessing the ``.selector`` member in an expression with side-effects (e.g., a function call), the expression would not be evaluated in the legacy code generation. Note that the code generated by the IR pipeline was unaffected and will behave as expected.",
```

Comment:
One more thing. This description does not properly mention the conditions needed to trigger this bug. This is important, because not every use of `.selector` is affected. Only the less sensible ones that no one would use in practice. Leaving it as is makes it sound potentially more likely to happen than it really is. We don't have to go into as much detail as in the blog post, but I think we should still describe when one may run into it more precisely.